libs/ledger-live-common/package.json

```diff
@@ -289,9 +295,8 @@
     "ts-jest": "^29.1.1",
     "ts-node": "^10.4.0",
     "typescript": "5.1.3",
+    "undici": "6.19.2",
     "uuid": "^8.3.2",
-    "ws": "7",
```

Comment:
Can we move back deps where they are on develop so we don't have useless changes